### PR TITLE
OSDOCS-9018: Added a link to the AWS docs for ROSA with HCP

### PR DESCRIPTION
**Pull request by EricPonvelle**
Version(s):
`enterprise-4.14+`

Issue:
[OSDOCS-9018](https://issues.redhat.com/browse/OSDOCS-9018)

Link to docs preview:
- [Creating ROSA with HCP clusters using the default options](https://file.rdu.redhat.com/eponvell/OSDOCS-9018_AWS-HCP-Link/rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.html)
    image

QE review:
- [ ] QE has approved this change.

Additional information:
Added a link to the AWS documentation about getting started with ROSA with HCP.

### CODE REVIEW

**Review by skrthomas — COMMENTED** [review]
/lgtm

**Review by wgordon17 — APPROVED** [review]
LGTM, thanks @EricPonvelle

**Review by kelbrown20 — COMMENTED** [review]
Hey @EricPonvelle! Just a small nit before merging. The link in this PR is not working for me, it says that "The web address you entered is not a functioning page on our site" on the webpage. Not sure if aws updated the paths or something

**Comment by wgordon17** [comment]
@kelbrown20, looks like is a temporary issue atm...it still shows up as a _cached_ result in google

File: rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc [hunk 1]
```diff
@@ -25,6 +25,9 @@ Since it is not possible to upgrade or convert existing ROSA clusters to a {hcp}
 {hcp-title} clusters only support AWS Security Token Service (STS) authentication.
 ====
 
+.Further reading
+* See the AWS documentation for information about link:https://docs.aws.amazon.com/ROSA/latest/userguide/getting-started-hcp.html[Getting started with ROSA with HCP using the ROSA CLI in auto mode].
```
Comment (wgordon17) [inline rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc:29]:
```suggestion
* See the AWS documentation for information about link:https://docs.aws.amazon.com/rosa/latest/userguide/getting-started-hcp.html[Getting started with ROSA with HCP using the ROSA CLI in auto mode].
```

**Review by wgordon17 — CHANGES_REQUESTED** [review]
My bad @kelbrown20, good catch! _Lower case_ `rosa` _does_ work 👍

**Review by wgordon17 — APPROVED** [review]
+1 (I can't believe it never dawned on me to actually _check_ the initial URL 🤦‍♂️)

**Review by kelbrown20 — COMMENTED** [review]
Updates LGTM!

**Pull request merged**